includes/class-amp-theme-support.php

```diff
@@ -523,6 +535,7 @@ public static function finish_output_buffering( $output ) {
 		$dom  = AMP_DOM_Utils::get_dom( $output );
 		$args = array(
 			'content_max_width' => ! empty( $content_width ) ? $content_width : AMP_Post_Template::CONTENT_MAX_WIDTH, // Back-compat.
+			'mutation_callback' => 'AMP_Mutation_Utils::track_removed',
```

Comment:
Per above, this should be conditional based on whether an authorized nonce is present.

Reply:
Good point. This is [now only added](https://github.com/Automattic/amp-wp/pull/912/files#diff-2585472f207548f626a43a7ff3cab922R797) if `AMP_Mutation_Utils::is_authorized()`. That checks for the nonce and capability.